Feature/workbox

danial24 opened the pull request:
## What I did
PWA via Workbox

artalat commented on packages/web/src/server/components/HTML/index.tsx:
```diff
@@ -35,10 +35,10 @@ const HTML = (props: HTMLProperties) => {
 						/>
 						<meta name="msapplication-TileColor" content="#2b2b2b" />
-						<meta
+						{/* <meta
```
if we dont need this, can we remove this code?

danial24 replied:
Yes will remove it after going through the usage of this file in detail.

artalat replied:
are we doing this?

danial24 commented:
@artalat 
There is only one check that is failing. Do let me know of this.
Please Review and Merge the PR.

artalat commented:
@danial24 where do we stand on this PR?

danial24 commented:
@artalat 
The develop branch was broken catering lint.
That is why it was failing.

There was no lint script in package.json in the expo package. Added lint and fix command there.
Fixed linting issues as well.
The PR is ready to merge

artalat commented:
@danial24 does this also resolve #48?

danial24 commented:
@artalat 
I dont think so. I guess we will have to fix it as well.

hashim-sohail commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [npm package (@latest dist-tag)](https://www.npmjs.com/package/@bluebase/cli-core)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

hashim-sohail commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [npm package (@latest dist-tag)](https://www.npmjs.com/package/@bluebase/cli)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
